### PR TITLE
added node js install tut

### PR DESCRIPTION
**Pull request by hardy613**
using nvm as a recommendation

### CODE REVIEW

**Comment by hardy613** [comment]
This is for one part of #51

**Comment by AGhost-7** [comment]
There was some stuff we did for youcompleteme, can you add that as well? We won't want to for the user to install rust as well though.

**Comment by shardy613** [comment]
sure, should I add it as `youcompleteme` is installed, or include installing `youcompleteme` as well?

**Comment by hardy613** [comment]
Whats the autocomplete engine you wanted `turn` or something?

**Comment by AGhost-7** [comment]
tern

**Comment by hardy613** [comment]
Tern info added. 

Cheers

File: tutorial/readme.md [hunk 1]
```diff
@@ -324,3 +324,66 @@ COPY ./.tmux.conf /tmp/.tmux.conf
...
+        "node": {}
+    }
+}
+```
+More info can be found in the 
+[tern documentation](http://ternjs.net/doc/manual.html).
+Now we need to copy and own the file. In the Dockerfile:
+```dockerfile
+# copy tern completion config
+COPY ./.tern-config "$HOME/.tern-config"
```
Comment (AGhost-7) [inline tutorial/readme.md]: Wouldn't it be easier to just tell the person to run this:

```
RUN echo '{"plugins": {"node": {}}}' > ~/.ternconfig
```

**Comment by hardy613** [comment]
Changes made, i was missing quite a bit :/

File: tutorial/readme.md (hunk 1 above)
Comment (hardy613) [reply tutorial/readme.md]: yes

**Pull request merged**